src/modules/api.js

```diff
@@ -1,8 +1,12 @@
 export default class Api {
   POKEMON_URL = 'https://pokeapi.co/api/v2/pokemon';
 
```

Comment:
The api breaks on newer browser. Please check out the branch called api-fix to view the fix. You can then merge and push again

Reply:
@matovu-farid thank's for the review. There is no difference between this branch and the api-fix branch

<img width="968" alt="Capture d’écran 2022-02-02 à 06 34 17" src="https://user-images.githubusercontent.com/7243132/152098525-f09c0fe0-1afd-4962-b138-581976f50392.png">